build/gen-make-frags/gen-make-frag.sh

```diff
@@ -443,11 +447,12 @@ main()
 
 
 	# Process our command line options.
-	while getopts ":dho:p:rv:" opt; do
+	while getopts ":dhi:o:p:rv:" opt; do
 		case $opt in
 			d  ) dry_run_flag="1" ;;
 			h  ) hide_flag="1" ;;
 			r  ) recursive_flag="1" ;;
+			i  ) ignore_list="$OPTARG" ;;
```

Comment:
This should probably be `ignore_list="$ignore_list $OPTARG"` to allow for multiple -i options.

Reply:
Good idea. I've made the change.